dcat/index.html

```diff
@@ -298,7 +298,7 @@ <h3>DCAT scope</h3>
               </p>
            </aside>
     <figure id="UML_DCAT_All_Attr">
-      <img alt="UML model of DCAT classes and properties" src="./images/DCAT-summary-all-attributes.png">
+      <img alt="UML model of DCAT classes and properties" src="./images/DCAT-summary-all-attributes.svg">
```

Comment:
```suggestion
      <img alt="UML model of DCAT classes and properties" src="./images/dcat-all-attributes.svg">
```

Reply:
Shouldn’t we change in uncapitalised letter also the name of the svg file, if we change it in the html?